teacher.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'person'
 class Teacher < Person
-  def initialize(specialization, name: 'Unknown')
-    super(name)
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
```

Comment:
- [][REQUIRED] I tested the teacher class and it doesn't seem to work. Please, check it once more.